lib/query-builders/admin.js

```diff
@@ -20,18 +22,34 @@ module.exports = {
   outstanding: (queryBuilder, profile) => {
     queryBuilder
       .where(buildEstablishmentQuery(profile))
-      .andWhere('status', returnedToApplicant.id);
+      .whereIn('status', [
+        returnedToApplicant.id,
+        recalledByApplicant.id
+      ]);
   },
 
   inProgress: (queryBuilder, profile) => {
     queryBuilder
       .where(buildEstablishmentQuery(profile))
-      .whereNotIn('status', [autoResolved.id, resolved.id, rejected.id, withdrawnByApplicant.id, returnedToApplicant.id]);
+      .whereNotIn('status', [
+        autoResolved.id,
+        resolved.id,
+        rejected.id,
+        withdrawnByApplicant.id,
+        returnedToApplicant.id,
+        recalledByApplicant.id,
```

Comment:
Recalled tasks are hidden from ASRU right? 👍

Reply:
Ah hang on this isn't ASRU this is establishment admin. Probably still makes sense though.

Reply:
recalled are seen by admins, but not asru